integration_tests/tests/billing-and-transaction-view-tests.ts

```diff
@@ -29,65 +32,65 @@ describe("\nUpload invoice to standardised folder and verify billing and transac
   };
   beforeAll(async () => {
     // tests are enabled to run sequentially as we are deleting the S3 directory in view tests so when running the test
-    // parallelly other tests will be interrupted(eg sns-s3 tests generate and checks eventId). We can enable to parallel once we implement BTM-340 the clean up for each test
-    await deleteDirectoryRecursiveInS3(bucketName, "btm_transactions");
+    // in parallel other tests will be interrupted(e.g. sns-s3 tests generate and checks eventId). We can enable to run in parallel
+    // once we implement BTM-340 to clean up after each test
+    await deleteS3Objects({ bucketName, prefix: "btm_transactions" });
     // uploading file to s3 will be removed once BTM-276 changes merged
     const file = "../payloads/receipt.txt";
     const filePath = path.join(__dirname, file);
-    const fileStream = fs.createReadStream(filePath);
-    await putObjectToS3(testObject, fileStream);
+    const fileData = fs.readFileSync(filePath);
+    await putS3Object({ target: testObject, data: fileData });
     const checkFileExists = await checkIfS3ObjectExists(testObject);
     expect(checkFileExists).toBe(true);
```

Comment:
I know this isn't part of the diff and would be one to address in another (smaller) PR but how do we feel about using `expect` like this?

Reply:
Well, I found many things I didn't like in the code I worked on and I could not fix everything. This is one of those things that I saw but didn't see as it needs fixing immediately, but I do agree.